webui/react/src/pages/TrialDetails/TrialDetailsHeader.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useMemo, useState } from 'react';
+
+import PageHeaderFoldable, { Option } from 'components/PageHeaderFoldable';
+import TrialHeaderLeft from 'pages/TrialDetails/Header/TrialHeaderLeft';
+import { Action, trialWillNeverHaveData } from 'pages/TrialDetails/TrialActions';
+import { openOrCreateTensorboard } from 'services/api';
+import { getStateColorCssVar } from 'themes';
+import { TrialDetails } from 'types';
+import { openCommand } from 'wait';
+
+interface Props {
+  fetchTrialDetails: () => void;
+  handleActionClick: (action: Action) => void;
+  trial: TrialDetails;
+}
+
+const TrialDetailsHeader: React.FC<Props> = (
+  { fetchTrialDetails, handleActionClick, trial }: Props,
+) => {
+  const [ isRunningTensorboard, setIsRunningTensorboard ] = useState<boolean>(false);
+
+  const headerOptions = useMemo<Option[]>(() => {
+    const options: Option[] = [];
+
+    if (trial.bestAvailableCheckpoint !== undefined) {
+      options.push({
+        key: Action.Continue,
+        label: 'Continue Trial',
```

Comment:
do we have an icon for this?

Reply:
It's a different variation of forking, so maybe just use the `fork` icon this too